.travis.yml

```diff
@@ -1,27 +1,36 @@
-language: python
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
+# The only current way of using OS X virtual machines in Travis CI
+# is setting up the environment for Objective-C
+# This is not a problem because Python will be installed through conda
+
+language: objective-c
```

Comment:
Hm, we'll have to watch out for that removal of `__init__.py` in the tests folder, I think I added it to have meaningful paths in the coverage reports:

https://github.com/poliastro/poliastro/commit/20153ca24482afb8624b5c9f61697e155a8a7332

-Five minutes later- Interestingly enough, I just went to your branch in coveralls and apparently everything is fine:

https://coveralls.io/builds/3057817

Let's see if we can merge the thing after your last commit!